lib/src/views/misc/circle_tab_indicator.dart

```diff
@@ -2,8 +2,9 @@ import 'package:flutter/material.dart';
 
 class CircleTabIndicator extends Decoration {
   final BoxPainter _painter;
+  final bool top;
 
-  CircleTabIndicator({required Color color}) : _painter = _CirclePainter(color);
+  CircleTabIndicator({required Color color, this.top = false}) : _painter = _CirclePainter(color, top);
```

Comment:
I would prefer these arguments to be on separate lines. We should try to enforce this in the formatter so it has to be consistent across the whole codebase.